File: packages/core/test/fixtures/symbols-polyglot/cart-csharp/checkout.cs
// Checkout — C# file-scoped-namespace + nested-type fixture (#1025).

namespace Cart.Checkout;

public class CheckoutService
{
    public int Total()
    {
        return 0;
    }

    public class Receipt
    {
        public string Render()
        {
            return "ok";
        }
    }
}

File: packages/core/test/fixtures/symbols-polyglot/fraud-kotlin/Checkout.kt
// Checkout — Kotlin second-package + nested-type fixture (#1034).

package com.example.checkout

class CheckoutService {
    fun total(): Int {
        return 0
    }

    class Receipt {
        fun render(): String {
            return "ok"
        }
    }
}

fun topLevelHelper(): Int = 42
